generators/app/index.js

```diff
@@ -315,14 +316,32 @@ module.exports = yeoman.Base.extend({
             { from: this.javaTemplateDir + '/web/rest/_JaversEntityAuditResource.java', to: this.javaDir + 'web/rest/JaversEntityAuditResource.java'}
           );
         }
+        if (this.enableTranslation) {
+          this.languages.forEach(function(language) {
+            var sourceLanguage = 'en';
+            if (fs.existsSync(this.templatePath() + '/src/main/webapp/i18n/' + language + '/entity-audit.json')) {
+              sourceLanguage = language;
+            }
+            files.push(
+              { from: this.webappDir + 'i18n/' + sourceLanguage + '/entity-audit.json', to: this.webappDir + 'i18n/' + language + '/entity-audit.json'}
+            );
+          }, this);
+        }
         this.copyFiles(files);
         // add bower dependency required
         jhipsterFunc.addBowerDependency('angular-object-diff', '1.0.3');
         jhipsterFunc.addAngularJsModule('ds.objectDiff');
         // add new menu entry
         jhipsterFunc.addElementToAdminMenu('entity-audit', 'list-alt', this.enableTranslation);
-        jhipsterFunc.addTranslationKeyToAllLanguages('entity-audit', 'Entity Audit', 'addAdminElementTranslationKey', this.enableTranslation);
```

Comment:
I tried to add complete i18n support. Do you see some drawback with this admin menu item translation implementation?